docs/_static/custom.css

```diff
@@ -4,8 +4,8 @@ div.cell_output table {
 }
 
 /*  png2x/retina rendering of figues in docs would also need to modify custom.css:
-https://github.com/spatialaudio/nbsphinx/issues/464#issuecomment-652729126  */
-.rst-content .image-reference img {
+based on https://github.com/spatialaudio/nbsphinx/issues/464#issuecomment-652729126  */
+div.cell_output img {
```

Comment:
this "works" for the large panel figures, but makes other long aspect ratio plots much too big. I don't want to spend too much time figuring out the correct settings, so I'm just goin to abandon customizing for retina figures and live with blurry labels in the docs

<img width="749" alt="image" src="https://github.com/user-attachments/assets/da9496b1-5710-4317-a99b-f2928c93727b" />